modules/mono/glue/GodotSharp/GodotSharp/Core/Attributes/GodotClassNameAttribute.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+namespace Godot
+{
+    /// <summary>
+    /// Attribute that specifies the engine class name when it's not the same
+    /// as the generated C# class name. This allows introspection code to find
+    /// the name associated with the class. If the attribute is not present,
+    /// the C# class name can be used instead.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class)]
+    public class GodotClassNameAttribute : Attribute
```

Comment:
I don't see this being added by the bindings generator. I understand this is still a WIP, but just in case it was an oversight.

Reply:
The attribute is already added by the bindings generator in master so I didn't have to do this in this PR. It just so happens that in master, since we always use the same name for classes that part of the code never executes.

https://github.com/godotengine/godot/blob/master/modules/mono/editor/bindings_generator.cpp#L1399-L1404